app/models/subnet.rb

```diff
@@ -193,7 +193,7 @@ def self.import(proxy)
 
       next if exists?(attrs)
       attrs.merge!(parse_dhcp_options(s['options'])) if s['options'].present?
-      new(attrs.update(:dhcp => proxy))
```

Comment:
Why not just adding `dhcp', 'dns' and `tftp` to attr_accessible? I think that'd be the only change required.